pkg/kubectl/cmd/exec/exec.go

```diff
@@ -159,32 +177,39 @@ func (p *ExecOptions) Complete(f cmdutil.Factory, cmd *cobra.Command, argsIn []s
 		}
 		p.Command = argsIn
 	} else {
-		p.PodName = argsIn[0]
+		p.ResourceName = argsIn[0]
 		p.Command = argsIn[1:]
-		if len(p.Command) < 1 {
```

Comment:
Why is this part removed?

Reply:
it is will validate in `Validate` method in this line https://github.com/kubernetes/kubernetes/blob/0c39d7d3806f1a45148f159b8480ab86df9f23ff/pkg/kubectl/cmd/exec/exec.go#L228-L230

i think this part is not necessary, so i leave it validated on the `Validate` method

Reply:
Makes sense, thanks!